lisbon-public-law-editions-full-note.csl

```diff
@@ -0,0 +1,1418 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" demote-non-dropping-particle="display-and-sort" page-range-format="chicago">
+  <info>
+    <title>Lisbon Public Law Editions - Full Note</title>
+    <title-short>LPL Editions Full</title-short>
+    <id>http://www.zotero.org/styles/lisbon-public-law-editions-full-note</id>
+    <link href="http://www.zotero.org/styles/lisbon-public-law-editions-full-note" rel="self"/>
```

Comment:
Please add the template style (presumably one of the Chicago ones) as well as the author guidelines. [See 8 and 10 here.](https://github.com/citation-style-language/styles/blob/master/STYLE_REQUIREMENTS.md)